ChangeLog

```diff
@@ -35,6 +35,7 @@ ver. 0.10.1-dev-1 (2016/??/??) - development edition
 -----------
 
 ### Fixes
+* fix nftables actions for IPv6 addresses (gh-1893)
```

Comment:
is not really a fix, rather an enhancement...

Reply:
Well with the current actions, I get errors. When I installed fail2ban 0.10.0, I got no warning that these actions didn't support IPv6 so I consider it to be a bug. However I can change it if you prefer.

Reply:
we've still many actions that are not IPv6-capable, but Ok...